tasks/PageIteratorTask.md

```diff
@@ -153,7 +153,19 @@ $pageIterator->iterate(function ($pageItem) use (&$items) {
       return true;
 });
 ```
+Python
+```
+page_iterator = PageIterator(response, client.request_adapter)
+page_iterator.set_headers({"Count": 2, "Skip": 2})
+
+page_result = page_iterator.convert_to_page(response)
+page_result.odata_next_link = "https://graph.microsoft.com/v1.0/users?$top=2&$skip=2"
+
+page = page_iterator.convert_to_page(page_result)
```

Comment:
```suggestion
```

I'm not sure why we'd have an extra step just for Python? also we weren't doing anything with this variable

Reply:
This looks to be called implictly in the `PageIterator` constructor. If the intent is to expose the `odata_next_link` I would suggest exposing it at a property accessor of the iterator itself which will hold the current NextLink being accessed.